### PR TITLE
Add Matrimony Keyboard

### PR DESCRIPTION
**Pull request by hypanim**
## Description
Adding the Matrimony Keyboard, which was sold in a group buy created by u/solciona on Reddit.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/matrimony/info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "keyboard_name": "Matrimony Keyboard",
+    "manufacturer": "Joshua Cheung",
+    "url": "https://www.reddit.com/r/mechmarket/comments/w1ytmc/gb_matrimony_split_keyboard/",
+    "maintainer": "hypanim",
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0x0001",
+        "device_version": "1.0.0"
+    }
+}
```
Comment (waffle87) [inline keyboards/matrimony/info.json:11]: This file needs the layout info. for the QMK Configurator. See [https://docs.qmk.fm/#/reference_configurator_support](https://docs.qmk.fm/#/reference_configurator_support).

File: keyboards/matrimony/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    KC_M,  KC_N,  KC_O,  KC_P,  KC_LSFT, KC_Z,    KC_X,    KC_C,     KC_V,   KC_B,   KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,  KC_UP,   MO(_FN1), KC_P1, KC_P2, KC_P3, \
+    KC_Q,  KC_R,  KC_S,  KC_T,  KC_LCTL, KC_LGUI, KC_LALT, MO(_FN1), KC_SPC, KC_SPC, MO(_FN1), KC_RALT, KC_RCTL, KC_LEFT, KC_DOWN, KC_RIGHT, KC_P0,   KC_PDOT,  KC_PENT),
+  [_FN1] = LAYOUT(
+    KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, \
+    KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, \
+    KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, \
+    KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, \
+    KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO)
```
Comment (waffle87) [inline keyboards/matrimony/keymaps/default/keymap.c:37]:
```suggestion
    KC_A,  KC_B,  KC_C,  KC_D,  QK_GESC, KC_1,    KC_2,    KC_3,     KC_4,   KC_5,   KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS,  KC_EQL,  KC_BSPC,  MO(_FN1),   KC_NUM,  KC_PSLS, KC_PAST, KC_PMNS,
    KC_E,  KC_F,  KC_G,  KC_H,  KC_TAB,  KC_Q,    KC_W,    KC_E,     KC_R,   KC_T,   KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC,  KC_RBRC, KC_BSLS,  MO(_FN1),   KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
    KC_I,  KC_J,  KC_K,  KC_L,  KC_CAPS, KC_A,    KC_S,    KC_D,     KC_F,   KC_G,   KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,  KC_ENT,  MO(_FN1), KC_P4, KC_P5, KC_P6,
    KC_M,  KC_N,  KC_O,  KC_P,  KC_LSFT, KC_Z,    KC_X,    KC_C,     KC_V,   KC_B,   KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,  KC_UP,   MO(_FN1), KC_P1, KC_P2, KC_P3,
    KC_Q,  KC_R,  KC_S,  KC_T,  KC_LCTL, KC_LGUI, KC_LALT, MO(_FN1), KC_SPC, KC_SPC, MO(_FN1), KC_RALT, KC_RCTL, KC_LEFT, KC_DOWN, KC_RIGHT, KC_P0,   KC_PDOT,  KC_PENT),
  [_FN1] = LAYOUT(
    KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_TRNS, KC_NO, KC_NO, KC_NO, KC_NO,
    KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_TRNS, KC_NO, KC_NO, KC_NO, KC_NO,
    KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_TRNS, KC_NO, KC_NO, KC_NO,
    KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_TRNS, KC_NO, KC_NO, KC_NO,
    KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_TRNS, KC_NO, KC_NO, KC_TRNS, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO)
```

File: keyboards/matrimony/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+#include QMK_KEYBOARD_H
+
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BASE 0
+#define _FN1 1
```
Comment (waffle87) [inline keyboards/matrimony/keymaps/default/keymap.c:23]:
```suggestion
enum layers {
    _BASE,
    _FN1
};
```

File: keyboards/matrimony/keymaps/left_only/keymap.c [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+[_FN1] = LAYOUT(
+    KC_PMNS,  KC_PAST, KC_PSLS, KC_NUM,     QK_GESC,   KC_1,    KC_2,    KC_3,     KC_4,   KC_5,   KC_6,      	        KC_7,    KC_8,    KC_9,     KC_0,    KC_MINS,  KC_EQL,  KC_BSPC,  MO(_FN1),   KC_NUM,  KC_PSLS, KC_PAST, KC_PMNS, \
+    KC_PPLS,  KC_P9,   KC_P8,   KC_P7,      KC_TAB,    KC_Q,    KC_W,    KC_E,     KC_R,   KC_T,   		KC_Y,   KC_U,    KC_I,    KC_O,     KC_P,    KC_LBRC,  KC_RBRC, KC_BSLS,  MO(_FN1),   KC_P7,   KC_P8,   KC_P9,   KC_PPLS, \
+    KC_P6,    KC_P5,   KC_P4,   KC_NO,      KC_CAPS,   KC_A,    KC_S,    KC_D,     KC_F,   KC_G,   			KC_H,    KC_J,    KC_K,     KC_L,    KC_SCLN,  KC_QUOT, KC_ENT,   MO(_FN1), 	       KC_P4,   KC_P5,   KC_P6, \
+    KC_P3,    KC_P2,   KC_P1,   KC_NO,      KC_LSFT,   KC_Z,    KC_X,    KC_C,     KC_V,   KC_B,   			KC_N,    KC_M,    KC_COMM,  KC_DOT,  KC_SLSH,  KC_RSFT, KC_UP,    MO(_FN1),            KC_P1,   KC_P2,   KC_P3, \
+    KC_PENT,  KC_PDOT, KC_P0,   KC_NO,      KC_LCTL,   KC_LGUI, KC_LALT, MO(_FN1), KC_SPC, 					 KC_SPC,  MO(_FN1), KC_RALT, KC_RCTL,  KC_LEFT, KC_DOWN,  KC_RIGHT, 	       KC_P0,   KC_PDOT, KC_PENT),
```
Comment (waffle87) [inline keyboards/matrimony/keymaps/left_only/keymap.c:37]: You need transparent keys (`KC_TRNS`) underneath all the momentary layer keys, as show in above suggestion.
```suggestion
    KC_A,  KC_B,  KC_C,  KC_D,  QK_GESC,   KC_1,    KC_2,    KC_3,     KC_4,   KC_5,   KC_6,      	KC_7,    KC_8,    KC_9,     KC_0,    KC_MINS,  KC_EQL,  KC_BSPC,  MO(_FN1),   KC_NUM,  KC_PSLS, KC_PAST, KC_PMNS,
    KC_E,  KC_F,  KC_G,  KC_H,  KC_TAB,    KC_Q,    KC_W,    KC_E,     KC_R,   KC_T,   		KC_Y,   KC_U,    KC_I,    KC_O,     KC_P,    KC_LBRC,  KC_RBRC, KC_BSLS,  MO(_FN1),   KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
    KC_I,  KC_J,  KC_K,  KC_L,  KC_CAPS,   KC_A,    KC_S,    KC_D,     KC_F,   KC_G,   			KC_H,    KC_J,    KC_K,     KC_L,    KC_SCLN,  KC_QUOT, KC_ENT,   MO(_FN1), 	       KC_P4,   KC_P5,   KC_P6,
    KC_M,  KC_N,  KC_O,  KC_P,  KC_LSFT,   KC_Z,    KC_X,    KC_C,     KC_V,   KC_B,   			KC_N,    KC_M,    KC_COMM,  KC_DOT,  KC_SLSH,  KC_RSFT, KC_UP,    MO(_FN1),            KC_P1,   KC_P2,   KC_P3,
    KC_Q,  KC_R,  KC_S,  KC_T,  KC_LCTL,   KC_LGUI, KC_LALT, MO(_FN1), KC_SPC, 					 KC_SPC,  MO(_FN1), KC_RALT, KC_RCTL,  KC_LEFT, KC_DOWN,  KC_RIGHT, 	       KC_P0,   KC_PDOT, KC_PENT),
[_FN1] = LAYOUT(
    KC_PMNS,  KC_PAST, KC_PSLS, KC_NUM,     QK_GESC,   KC_1,    KC_2,    KC_3,     KC_4,   KC_5,   KC_6,      	        KC_7,    KC_8,    KC_9,     KC_0,    KC_MINS,  KC_EQL,  KC_BSPC,  MO(_FN1),   KC_NUM,  KC_PSLS, KC_PAST, KC_PMNS,
    KC_PPLS,  KC_P9,   KC_P8,   KC_P7,      KC_TAB,    KC_Q,    KC_W,    KC_E,     KC_R,   KC_T,   		KC_Y,   KC_U,    KC_I,    KC_O,     KC_P,    KC_LBRC,  KC_RBRC, KC_BSLS,  MO(_FN1),   KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
    KC_P6,    KC_P5,   KC_P4,   KC_NO,      KC_CAPS,   KC_A,    KC_S,    KC_D,     KC_F,   KC_G,   			KC_H,    KC_J,    KC_K,     KC_L,    KC_SCLN,  KC_QUOT, KC_ENT,   MO(_FN1), 	       KC_P4,   KC_P5,   KC_P6,
    KC_P3,    KC_P2,   KC_P1,   KC_NO,      KC_LSFT,   KC_Z,    KC_X,    KC_C,     KC_V,   KC_B,   			KC_N,    KC_M,    KC_COMM,  KC_DOT,  KC_SLSH,  KC_RSFT, KC_UP,    MO(_FN1),            KC_P1,   KC_P2,   KC_P3,
    KC_PENT,  KC_PDOT, KC_P0,   KC_NO,      KC_LCTL,   KC_LGUI, KC_LALT, MO(_FN1), KC_SPC, 					 KC_SPC,  MO(_FN1), KC_RALT, KC_RCTL,  KC_LEFT, KC_DOWN,  KC_RIGHT, 	       KC_P0,   KC_PDOT, KC_PENT),
```

File: keyboards/matrimony/readme.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Matrimony
+
+* Keyboard Maintainer: [Joshua Cheung](https://github.com/hypanim)
+* Hardware Supported: Elite-C
+* Hardware Availability: https://www.reddit.com/r/mechmarket/comments/w1ytmc/gb_matrimony_split_keyboard/?utm_source=share&utm_medium=web2x&context=3
+
+Both sides of the split keyboard are flashed with the same firmware. Make sure to set the eeprom of each side to their respective left/right configuration using QMK toolbox before flashing. The BootMagic key for the left side is the top left, while the BootMagic key for the right side is the top right. 
+```
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
Comment (waffle87) [inline keyboards/matrimony/readme.md:9]: Please add instructions on entering bootloader mode. 
eg.
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available

File: keyboards/matrimony/keymaps/left_only/keymap.c [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+#include QMK_KEYBOARD_H
+
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BASE 0
+#define _FN1 1
```
Comment (waffle87) [inline keyboards/matrimony/keymaps/left_only/keymap.c:23]:
```suggestion
enum layers {
    _BASE,
    _FN1
};
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/matrimony/readme.md (hunk 5 above)
Comment (hypanim) [reply keyboards/matrimony/readme.md:9]: Thank you so much for your help! I'm currently putting all these changes together.

**Comment by hypanim** [comment]
Changes are added!

File: keyboards/matrimony/readme.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+* Hardware Supported: Elite-C
+* Hardware Availability: https://www.reddit.com/r/mechmarket/comments/w1ytmc/gb_matrimony_split_keyboard/?utm_source=share&utm_medium=web2x&context=3
+
+Both sides of the split keyboard are flashed with the same firmware. Make sure to set the eeprom of each side to their respective left/right configuration using QMK toolbox before flashing.
+
+Enter the bootloader in one of 2 ways:
+
+Bootmagic reset: For the left side, hold down the top-left key and plug in the keyboard. For the right side, hold down the top right key and plug in the keyboard.
+Physical reset button: Briefly press the button on the Elite-C
+```
```
Comment (drashna) [inline keyboards/matrimony/readme.md:13]:
```suggestion
* Bootmagic reset: For the left side, hold down the top-left key and plug in the keyboard. For the right side, hold down the top right key and plug in the keyboard.
* Physical reset button: Briefly press the button on the Elite-C

```

File: keyboards/matrimony/rules.mk [hunk 8]
```diff
@@ -0,0 +1,21 @@
+# MCU name
+MCU = atmega32u4
+BOOTLOADER = atmel-dfu
+
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = no       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
```
Comment (drashna) [inline keyboards/matrimony/rules.mk:10]:
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

File: keyboards/matrimony/rules.mk [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
+OLED_DRIVER_ENABLE = no	  # prevents lag
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
```
Comment (drashna) [inline keyboards/matrimony/rules.mk:19]:
```suggestion
AUDIO_ENABLE = no           # Audio output
RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by hypanim** [comment]
Fixed! Thanks to everyone for their input!

**Review by drashna — APPROVED** [review]

File: keyboards/matrimony/rules.mk [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = no       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+AUDIO_ENABLE = no           # Audio output
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.
+
```
Comment (fauxpark) [inline keyboards/matrimony/rules.mk:14]:
```suggestion

# Bootloader selection
BOOTLOADER = atmel-dfu

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = no        # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = no            # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/matrimony/config.h [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+#define BOOTMAGIC_LITE_COLUMN 4
+#define BOOTMAGIC_LITE_ROW_RIGHT 5
+#define BOOTMAGIC_LITE_COLUMN_RIGHT 8
+
+/* key matrix size */
+// Rows are doubled-up
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 13
+
+
```
Comment (fauxpark) [inline keyboards/matrimony/config.h:30]:
```suggestion
```

File: keyboards/matrimony/config.h [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+#define DIODE_DIRECTION COL2ROW
+
+#define MATRIX_ROW_PINS { D4, C6, D7, E6, B4 }
+// B7 and B5 are dummy pins to ensure both columns == 13
+#define MATRIX_COL_PINS { D2, D3, B5, B6, F4, F5, F6, F7, B1, B3, B2, B7, D5 }
+#define MATRIX_ROW_PINS_RIGHT { F0, F1, C7, D5, B7 }
+#define MATRIX_COL_PINS_RIGHT { B6, B2, B3, B1, F7, F6, F5, F4, B0, B5, B4, E6, D7 }
+
+// Set 0 if debouncing isn't needed
+#define DEBOUNCE 5
```
Comment (fauxpark) [inline keyboards/matrimony/config.h:41]:
```suggestion
```

File: keyboards/matrimony/rules.mk [hunk 13]
```diff
@@ -0,0 +1,19 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
```
Comment (fauxpark) [inline keyboards/matrimony/rules.mk:6]:
```suggestion
```

File: keyboards/matrimony/info.json [hunk 14]
```diff
@@ -0,0 +1,567 @@
+{
+    "keyboard_name": "Matrimony Keyboard",
+    "manufacturer": "Joshua Cheung",
+    "url": "https://www.reddit.com/r/mechmarket/comments/w1ytmc/gb_matrimony_split_keyboard/",
+    "maintainer": "hypanim",
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0x0001",
+        "device_version": "1.0.0"
+    },
```
Comment (fauxpark) [inline keyboards/matrimony/info.json:10]:
```suggestion
    },
    "processor": "atmega32u4",
    "bootloader": "atmel-dfu",
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/matrimony/config.h (hunk 11 above)
Comment (hypanim) [reply keyboards/matrimony/config.h:30]: These lines are required for my keyboard to work I think, I tried flashing without, and it doesn't work.

**Comment by hypanim** [comment]
Hello, I'm curious as to what changes are needed for this merge to be approved.

File: keyboards/matrimony/config.h (hunk 11 above)
Comment (fauxpark) [reply keyboards/matrimony/config.h:30]: See #19581

File: keyboards/matrimony/readme.md [hunk 15]
```diff
@@ -0,0 +1,14 @@
...
+* Hardware Supported: Elite-C
+* Hardware Availability: https://www.reddit.com/r/mechmarket/comments/w1ytmc/gb_matrimony_split_keyboard/?utm_source=share&utm_medium=web2x&context=3
+
+Both sides of the split keyboard are flashed with the same firmware. Make sure to set the eeprom of each side to their respective left/right configuration using QMK toolbox before flashing.
+
+Enter the bootloader in one of 2 ways:
+
+* Bootmagic reset: For the left side, hold down the top-left key and plug in the keyboard. For the right side, hold down the top right key and plug in the keyboard.
+* Physical reset button: Briefly press the button on the Elite-C
+
```
Comment (waffle87) [inline keyboards/matrimony/readme.md:13]:
```suggestion

Make example for this keyboard (after setting up your build environment):

    make matrimony:default

Flashing example for this keyboard:

    make matrimony:default:flash

```

File: keyboards/matrimony/keymaps/default/readme.md [hunk 16]
```diff
@@ -0,0 +1 @@
+# The default keymap for matrimony
```
Comment (waffle87) [inline keyboards/matrimony/keymaps/default/readme.md:1]: This file should be removed.

**Review by waffle87 — CHANGES_REQUESTED** [review]
Fauxpark's suggestions also need applying.

**Pull request closed without merging**